src/Order/OrderTests/APITests.cs

```diff
@@ -0,0 +1,246 @@
+using API.Abstraction.Helpers;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Moq;
+using OrderAPI.Controllers;
+using OrderAPI.DTOs;
+using OrderApplication.Commands;
+using OrderApplication.Entities;
+using OrderApplication.Queries;
+using TestsCore;
+
+namespace OrderTests.API
+{
+	public class APITests
+	{
+		[Fact]
+		public async Task GetDetails_ResultNull_NotFound()
+		{
+			var mediator = new Mock<IMediator>();
+			mediator.Setup(m => m.Send(It.IsAny<GetCartQuery>(), It.IsAny<CancellationToken>()))
+				.ReturnsAsync((CartDetails)null);
```

Comment:
this initialization looks similar in other tests but with different result, can be moved to a separate method, like `SetupMediator(bool? withSuccessResult = null)...` 
Also linkGenerator can be moved to private fields and initialized in constructor.

Reply:
Was moved to `TestCore` along with controller's initialization.